webpack/assets/javascripts/react_app/components/HostDetails/Templates/CardItem/CardTemplate/index.js

```diff
@@ -25,7 +25,8 @@ const CardTemplate = ({
   const { cardExpandStates, dispatch, registerCard } = useContext(
     CardExpansionContext
   );
-  const cardId = header;
+  const cardId =
+    typeof header === 'string' || header instanceof String ? header : ouiaId;
```

Comment:
I don't think `instanceof` is needed. `typeof` is enough.

Reply:
`instanceof` seems to have weird behavior anyway:
![image](https://github.com/user-attachments/assets/5390e8e7-7835-4958-8c22-b83faa3328fc)

Reply:
String literal is not an object. Only `new String("something")` would be `instance of String`.